plankton/scripts/run.py

```diff
@@ -61,23 +62,23 @@
                     help='Arguments for the adapter.')
 
 
-def run_simulation(argument_list=None):
+def do_run_simulation(argument_list=None):
     arguments = parser.parse_args(argument_list or sys.argv[1:])
 
     if arguments.version:
-        print(__version__)
-        return
+        return __version__
```

Comment:
I've thought further about this. It looks like we're on a path towards separating the construction of the simulation from running it as well. This can be useful for running simulations in a different context (you mentioned a service in one of the issues). The context might not support printing, but it could still start from using this function and evaluating its return value.